apps/ai_reports/tasks.py

```diff
@@ -0,0 +1,57 @@
+import ast
+import logging
+
+import backoff
+import httpx
+from celery import shared_task
+from django.conf import settings
+
+from adhocracy4.comments.models import Comment
+from apps.ai_reports.models import AiReport
+
+client = httpx.Client()
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def get_classification_for_comment(comment_pk: int) -> None:
+    try:
+        comment = Comment.objects.get(pk=comment_pk)
+        response = call_ai_api(comment=comment)
+        if response.status_code == 200:
+            extract_and_save_ai_classifications(comment=comment, report=response.json())
```

Comment:
what other response code can give? Good to add an else statement here and logger warning